### PR TITLE
Fix Workflow URL Bug

### PR DESCRIPTION
**Pull request by Cwagne17**
Fixes url, completes [Story 8511](https://app.shortcut.com/clarkcan/story/8511/patch-actions-workflow-in-github-repositories)

### CODE REVIEW

File: src/README.md [hunk 1]
```diff
@@ -1,26 +1,41 @@
...
+--->
 
-The default `.circleci/config.yml` file contains the configuration code needed to automatically pack, test, and deploy and changes made to the contents of the orb source in this directory.
+A starter template for orb projects. Build, test, and publish orbs automatically on CircleCI with [Orb-Tools](https://circleci.com/orbs/registry/orb/circleci/orb-tools).
 
-## @orb.yml
+Additional READMEs are available in each directory.
 
-This is the entry point for our orb "tree", which becomes our `orb.yml` file later.
+**Meta**: This repository is open for contributions! Feel free to open a pull request with your changes. Due to the nature of this repository, it is not built on CircleCI. The Resources and How to Contribute sections relate to an orb created with this template, rather than the template itself.
```
Comment (EVogel1999) [inline src/README.md:11]: Meta?

**Review by EVogel1999 — COMMENTED** [review]
Just one question.  Looks good!

**Review by StarChaser44 — APPROVED** [review]

**Review by JosueMolinaMorales — APPROVED** [review]

**Review by mfranz1 — APPROVED** [review]

File: src/README.md (hunk 1 above)
Comment (pzalep1) [reply src/README.md:11]: ^

Comment (Cwagne17) [reply src/README.md:11]: This is just the default README.md packaged with it

**Review by pzalep1 — APPROVED** [review]

**Review by EVogel1999 — APPROVED** [review]

**Review by StarChaser44 — APPROVED** [review]

File: src/README.md [hunk 2]
```diff
@@ -1,26 +1,40 @@
...
- - [Orb Author Intro](https://circleci.com/docs/2.0/orb-author-intro/#section=configuration)
- - [Reusable Configuration](https://circleci.com/docs/2.0/reusing-config)
\ No newline at end of file
+We welcome [issues](https://github.com/<organization>/<project-name>/issues) and [pull requests](https://github.com/<organization>/<project-name>/pulls) against this repository!
+
+### How to Publish
+
+* Create and push a branch with your new features.
+* When ready to publish a new production version, create a Pull Request from _feature branch_ to `master`.
+* The title of the pull request must contain a special semver tag: `[semver:<segment>]` where `<segment>` is replaced by one of the following values.
```
Comment (pzalep1) [inline src/README.md:26]: Where did you get this template? I like it.

**Review by pzalep1 — APPROVED** [review]

**Review by JosueMolinaMorales — APPROVED** [review]

**Review by mfranz1 — APPROVED** [review]

**Pull request merged**